README.md

```diff
@@ -27,51 +31,49 @@ The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool
 | [Squid](https://github.com/Kingsford-Group/squid)                         |         :x:        |   `1.5`  |
 | [Star-Fusion](https://github.com/STAR-Fusion/STAR-Fusion)                 | :white_check_mark: |  `1.9.1` |
 
+## Pipeline summary
+
+<!-- TODO nf-core: Fill in short bullet-pointed list of the default steps in the pipeline -->
+
+1. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
+2. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
```

Comment:
I think we have a better summary

Reply:
It definitely needs to be more detailed, but the current README corresponds to the current workflow, which is basically a placeholder. Would it be fine to write the detailed steps when I have the rnafusion workflow ready?